packages/react/src/components/FileUploader/Filename.tsx

```diff
@@ -31,10 +67,15 @@ function Filename({
           {invalid && <WarningFilled className={`${prefix}--file-invalid`} />}
           <button
             aria-label={`${iconDescription} - ${name}`}
-            aria-describedby={invalid ? ariaDescribedBy : null}
             className={`${prefix}--file-close`}
             type="button"
-            {...rest}>
+            {...rest}
+            tabIndex={
+              rest.tabIndex !== undefined
+                ? parseInt(rest.tabIndex as string, 10)
+                : undefined
+            }
+            aria-describedby={invalid ? ariaDescribedBy : undefined}>
```

Comment:
Sure thing. I just want to note here incase you missed it - this prop used to be called `aria-describedBy` previously, but now is changed to match the actual HTML attribute `aria-describedby` (no caps). 

\- `aria-describedBy`
\+ `aria-describedby`

Moving this above `...rest` will not really do anything though, since `rest` does not contain `aria-describedby`. In either case, the user can't overwrite the conditional behaviour, it seems to me.

Reply:
Ah! Yes I missed that. Sorry my mistake 🤦‍♂️